openedx/features/caliper_tracking/transformers/course_transformers.py

```diff
@@ -0,0 +1,15 @@
+"""
```

Comment:
rename this file `enrollment_transformers.py`

Reply:
done in the latest commit